etc/Build.sh

```diff
@@ -72,6 +73,9 @@ while [ "$#" -gt 0 ]; do
         -no-gui)
             cmakeOptions+=" -DBUILD_GUI=OFF"
             ;;
+        -build-man)
+            cmakeOptions+=" -DBUILD_MAN=ON"
+            ;;
```

Comment:
How long does this take?  I'm wondering if it should be the default.

Reply:
About 2-3 mins with `-j16` if I recall correctly.